src/main/java/jdbchomework/utils/InjectorJdbc.java

```diff
@@ -8,11 +8,13 @@
 import jdbchomework.controller.DeveloperController;
 import jdbchomework.controller.ProjectController;
 import jdbchomework.controller.SkillController;
+import jdbchomework.dao.hibernate.CompaniesHibDao;
 import jdbchomework.dao.jdbc.CompaniesJdbcDao;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'jdbchomework.dao.jdbc.CompaniesJdbcDao'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)